lager/lenses.hpp

```diff
@@ -103,15 +224,10 @@ namespace lenses {
 //! @{
 
 template <typename Getter, typename Setter>
-auto getset(Getter&& getter, Setter&& setter)
+auto getset(const Getter& getter, const Setter& setter)
```

Comment:
Just a note: the change to `const Getter& getter` is not a regression, because they were previously bound to a const reference inside a const (immutable) lambda.